README.md

```diff
@@ -13,3 +9,66 @@ Make sure that you have the compatible versions of Visual Studio 2019 and .NET C
+
+
+[sales@syncfusion.com](mailto:sales@syncfusion.com?Subject=Syncfusion Blazor Menu Bar - GitHub) | [www.syncfusion.com](https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-menu-bar-github-samples) | Toll Free: 1-888-9 DOTNET
```

Comment:
change the footer
![image](https://user-images.githubusercontent.com/61148953/161212607-d2877697-3c96-4c08-9b20-14c66b91d8f6.png)